src/responses/friendship.repository.change.response.ts

```diff
@@ -1,8 +1,12 @@
 export interface FriendshipRepositoryChangeResponseRootObject {
-  friendship_status: FriendshipRepositoryChangeResponseFriendship_status;
+  friendship_status: FriendshipRepositoryResponseFriendship_status;
   status: string;
 }
-export interface FriendshipRepositoryChangeResponseFriendship_status {
+export interface FriendshipBestiesRepositoryResponseRootObject {
```

Comment:
This should be named `FriendshipRepositorySetBestiesResponseRootObject` and live in its own file.